docs/pom.xml

```diff
@@ -3525,6 +3523,16 @@
                             </execution>
                         </executions>
                     </plugin>
+                    <plugin>
+                        <groupId>us.bryon</groupId>
+                        <artifactId>graphviz-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>dot-files</id>
+                                <phase>none</phase>
+                            </execution>
+                        </executions>
+                    </plugin>
```

Comment:
Why do we need this?

Reply:
There is no skip config for graphviz. This is how you can disable the run. This was pointed to me by @ppalaga a while ago.